test/core/fixtures/transformation/validation.undeclared-variable-check/undeclared-shorthand-property/exec.js

```diff
@@ -0,0 +1 @@
+({foo})
```

Comment:
This test should pass without the `shorthand` addition anyway since `value` will be getting visited on the `Property` node.